src/NFTBill.sol

```diff
@@ -13,11 +13,11 @@ contract NFTBill is ERC1155 {
     }
 
     function deposit() external payable {
-        require(msg.value > 0, 'Send at least 1 wei');
-        require(msg.value <= type(uint96).max, 'Too much ETH');
+        uint256 msgValue = msg.value;
```

Comment:
how about naming it `value` for consistency with erc20 deposit below?

Reply:
good idea. done